e2e/e2e_test.go

```diff
@@ -41,27 +47,86 @@ import (
 )
 
 var (
-	amount            int64 = 1000000000
-	hBarSendAmount          = hedera.HbarFromTinybar(amount)
-	hbarRemovalAmount       = hedera.HbarFromTinybar(-amount)
+	receiveAmount     uint64 = 100
+	tinyBarAmount     int64  = 1000000000
+	hBarSendAmount           = hedera.HbarFromTinybar(tinyBarAmount)
+	hbarRemovalAmount        = hedera.HbarFromTinybar(-tinyBarAmount)
+	now                      = time.Now()
 )
 
 const (
 	expectedValidatorsCount = 3
 )
 
+func Test_Ethereum_Hedera_HBAR(t *testing.T) {
+	setupEnv := setup.Load()
+	now = time.Now()
+	accountBalanceBefore := util.GetHederaAccountBalance(setupEnv.Clients.Hedera, setupEnv.Clients.Hedera.GetOperatorAccountID(), t)
+
+	// 1. Submit burn transaction to the bridge contract
+	submittedTx, expectedRouterBurn := sendEthTransaction(setupEnv, constants.Hbar, t)
+
+	// 2. Validate burn event
+	expectedId := validateBurnEvent(setupEnv, submittedTx, expectedRouterBurn, t)
+
+	// 3. Validate that the tx went through and emitted the correct events
+	mirrorNodeScheduledTransaction := validateScheduledTx(setupEnv, t)
+
+	// 4. Validate that the balance of the receiver account (hedera) was changed with the correct amount
+	validateReceiverAccountBalance(setupEnv, accountBalanceBefore, constants.Hbar, t)
+
+	// 5. Prepare Expected Database Record
+	expectedBurnEventRecord := util.PrepareExpectedBurnEventRecord(
+		mirrorNodeScheduledTransaction.TransactionID,
+		int64(receiveAmount),
+		setupEnv.Clients.Hedera.GetOperatorAccountID(),
+		expectedId)
+
+	// 6. Validate Database Record
+	verifyBurnEventRecord(setupEnv.DbValidator, expectedBurnEventRecord, t)
+}
+
+func Test_Ethereum_Hedera_Token(t *testing.T) {
+	setupEnv := setup.Load()
+	now = time.Now()
+	accountBalanceBefore := util.GetHederaAccountBalance(setupEnv.Clients.Hedera, setupEnv.Clients.Hedera.GetOperatorAccountID(), t)
+
+	// 1. Submit burn transaction to the bridge contract
+	submittedTx, expectedRouterBurn := sendEthTransaction(setupEnv, constants.Token, t)
+
+	// 2. Validate burn event
+	expectedId := validateBurnEvent(setupEnv, submittedTx, expectedRouterBurn, t)
+
+	// 3. Validate that the tx went through and emitted the correct events
```

Comment:
This inline comment is not correct. It seems to refer to step 2 "validateBurnEvent"